pyproject.toml

```diff
@@ -141,7 +142,7 @@ scanorama = ["scanorama"]  # Scanorama dataset integration
 scrublet = ["scikit-image"]  # Doublet detection with automatic thresholds
 # Acceleration
 rapids = ["cudf>=0.9", "cuml>=0.9", "cugraph>=0.9"]  # GPU accelerated calculation of neighbors
-dask = ["dask[array]!=2.17.0"]  # Use the Dask parallelization engine
+dask = ["dask[array]>=2022.09"]  # Use the Dask parallelization engine
```

Comment:
this is also calver, but they *do* issue multiple feature releases per month sometimes

Reply:
TODO: specify the day

Reply:
```suggestion
dask = ["dask[array]>=2022.09.3"]  # Use the Dask parallelization engine
```